packages/rgbpp/src/rgbpp/utils/transaction.ts

```diff
@@ -0,0 +1,61 @@
+import { BaseCkbVirtualTxResult } from '@rgbpp-sdk/ckb';
+import { BtcAssetsApi } from '@rgbpp-sdk/service';
+
+export interface RgbppTxGroup {
+  ckbVirtualTxResult: BaseCkbVirtualTxResult | string;
+  btcTxHex: string;
+}
+
+export interface SentRgbppTxGroup {
+  btc: {
+    sent: boolean;
+    txId?: string;
+    error?: Error | unknown;
+  };
+  ckb: {
+    sent: boolean;
+    error?: Error | unknown;
+  };
+}
+
+export async function sendRgbppTxGroups(props: {
+  txGroups: RgbppTxGroup[];
+  btcService: BtcAssetsApi;
+  retry?: number;
+}): Promise<SentRgbppTxGroup[]> {
+  const results: SentRgbppTxGroup[] = [];
+  for (const group of props.txGroups) {
+    const result: SentRgbppTxGroup = {
+      ckb: {
+        sent: false,
+      },
+      btc: {
+        sent: false,
+      },
+    };
+    try {
+      // TODO: add retry logic
```

Comment:
Is there any logic to be implemented?

Reply:
The retry logic is not implemented yet. I was not sure about the feature, so I left a TODO comment.

Reply:
The "retry" prop is removed for now at 37074c4